src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -103,7 +103,7 @@ public final class InlineConfigParser {
             "com.puppycrawl.tools.checkstyle.checks.regexp.RegexpCheck",
             "com.puppycrawl.tools.checkstyle.checks.whitespace.EmptyForInitializerPadCheck",
             "com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocMethodCheck",
-            "com.puppycrawl.tools.checkstyle.checks.modifier.ModifierOrderCheck",
+            "com.puppycrawl.tools.checkstyle.checks.annotation.MissingDeprecatedCheck",
```

Comment:
@coderdeadpool Why do we not have simple removal of check from this set?

Reply:
Should be simple removal

Reply:
@nrmancuso @romani I have simply removed this line . However it is showing some conflicts ,  So i rebase the branch and resolved the conflicts.

Reply:
Conflict was not resolved correctly, this line needs to be completely removed.

Reply:
Done.